src/pages/proposals/ProposalList.vue

```diff
@@ -475,12 +497,41 @@ q-page.page-proposals
       :viewSelectorLabel="'View'",
       :chipsFiltersLabel="'Proposal types'",
       :filters.sync="filters"
-      :toggleLabel="'Show Staging Proposals'"
+      :toggleLabel="'Staging Proposals'"
       :toggle.sync="showStagedProposals",
       :toggleDefault="true"
       :showToggle="true",
       :style="mobileFilterStyles"
+      :filterTitle="'Search proposals'"
+      :showViewSelector="false"
       )
+      .col
+        base-placeholder.q-mr-sm(v-if="!filteredProposals.length && !filteredStagedProposals.length && !$apollo.loading" title= "No Proposals" subtitle="Your organization has not created any proposals yet. You can create a new proposal by clicking the button below."
```

Comment:
@evgenibir It is not really readable like this. Put each property in a new line, like:
```vue
...
title="No Proposals"
subtitle="Your organization has not created..."
...
```

Reply:
fixed